src/uproot/compression.py

```diff
@@ -88,11 +88,15 @@ class _DecompressZLIB:
     name = "ZLIB"
     _2byte = b"ZL"
     _method = b"\x08"
+    zlib = uproot.extras.zlib()
 
     def decompress(self, data: bytes, uncompressed_bytes=None) -> bytes:
-        import zlib
-
-        return zlib.decompress(data)
+        zlib = self.zlib
```

Comment:
The only reason it was imported like this was for symmetry with the other libraries.